src/js/state/Packets/flows.js

```diff
@@ -38,6 +38,7 @@ export default {
       })
       .catch((error) => {
         dispatch(Packets.error(log.getString("uid"), error))
+        throw error
```

Comment:
This catch was swallowing the error, which resulted in trying to open an undefined file here: https://github.com/brimsec/brim/blob/surface-download-errors/src/js/flows/downloadPcap.js#L9